flake.nix

```diff
@@ -3,7 +3,7 @@
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
     nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
     nixpkgs-kitman.url = "github:jkitman/nixpkgs/add-esplora-pkg";
-    crane.url = "github:ipetkov/crane?ref=master&ref=953b70da2813fb882c39890f2514e7db76fc8843";
+    crane.url = "github:ipetkov/crane?ref=master&ref=116b32c30b5ff28e49f4fcbeeb1bbe3544593204";
```

Comment:
are these both meant to be `ref` shouldn't open be `rev`

Reply:
Right.